rqt_bag/src/rqt_bag/bag_timeline.py

```diff
@@ -752,12 +781,26 @@ def toggle_play(self):
 
     def navigate_play(self):
         self.play_speed = 1.0
+        self.last_frame = rospy.Time.from_sec(time.time())
+        self.last_playhead = self._timeline_frame.playhead
         self._play_timer.start()
 
     def navigate_stop(self):
         self.play_speed = 0.0
         self._play_timer.stop()
 
+    def navigate_previous(self):
+        self.navigate_stop()
+        self.play_speed = -1.0
```

Comment:
Why are you overriding the navigate_stop play_speed here?

If the button is intended to step through the timesteps the play_speed should be 0.0 after pressed. 
I think this line should simply be removed. I tested it without it and it still works.